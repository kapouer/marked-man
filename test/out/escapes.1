()                                                                          ()

NNAAMMEE
       escapes tests

       backslashes before the asterisks, like this:
         iteral asterisks

       Markdown provides backslash escapes for the following characters:
            backslash
         `   backtick
         *   asterisk
         _   underscore
         {}  curly braces
         []  square brackets
         ()  parentheses
         #   hash mark
         +   plus sign
         -   minus sign (hyphen)
         !   exclamation mark

                                 November 2022                              ()
