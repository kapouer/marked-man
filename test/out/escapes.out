[4mESCAPES[24m(1)                  General Commands Manual                 [4mESCAPES[24m(1)

[1mNAME[0m
       [1mescapes[0m

       backslashes before the asterisks, like this:
         \*literal asterisks\*

       Markdown provides backslash escapes for the following characters:
         \   backslash
         â€˜   backtick
         *   asterisk
         _   underscore
         {}  curly braces
         []  square brackets
         ()  parentheses
         #   hash mark
         +   plus sign
         -   minus sign (hyphen)
         .   dot
         !   exclamation mark

                                 January 1979                       [4mESCAPES[24m(1)
